src/templates/xpath_validation_errors.html

```diff
@@ -1,10 +1,7 @@
 <div class="alert alert-danger">
     <h4><i class="fa fa-warning"></i> Validation Failed</h4>
-    <p>Please fix all errors before leaving this page:</p>
-    <%=errors%>
-    <ul>
-    <% _.each(errors, function (error) { %>
-        <li><%=error%></li>
-    <% }); %>
```

Comment:
It listed them twice. You can see it in the screenshot I posted. I'm not actually sure when it was valid to have multiple errors. The parser will only return one